src/ad-bidders/prebid/adapters/index-exchange.ts

```diff
@@ -12,6 +12,42 @@ export class IndexExchange extends PrebidAdapter {
 	}
 
 	prepareConfigForAdUnit(code, { sizes, siteId }): PrebidAdUnit {
+		switch (code.toLowerCase()) {
```

Comment:
I know that we already did it in a few adapters, but in my opinion, we shouldn't use hardcoded (non-general) values here. What about setting an appropriate flag in the slot context and then:
```suggestion
		if (context.get(`slots.${code}.isVideo`)) {
			return this.getVideoConfig(code, siteId);
		}

		return this.getStandardConfig(code, { sizes, siteId });
```